platform-api/che-core-api-git/src/test/java/org/eclipse/che/git/impl/LsRemoteTest.java

```diff
@@ -68,7 +68,7 @@ public void testShouldBeAbleToGetResultFromPublicRepo(GitConnectionFactory conne
 
     @Test(dataProvider = "GitConnectionFactory", dataProviderClass = org.eclipse.che.git.impl.GitConnectionFactoryProvider.class,
           expectedExceptions = UnauthorizedException.class,
-          expectedExceptionsMessageRegExp = "Not authorized")
+          expectedExceptionsMessageRegExp = "fatal: Authentication failed for 'https://bitbucket.org/exoinvitemain/privater.git/'\n")
```

Comment:
For executing of command is always used english locale https://github.com/codenvy/che-core/blob/master/che-core-git-impl-native/src/main/java/org/eclipse/che/git/impl/nativegit/CommandProcess.java#L54